engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -342,11 +342,16 @@ class WorkflowManagerActor(config: Config)
 
     import PromiseActor.EnhancedActorRef
 
+    val shadowExecutionEnabled = config.getBoolean("system.shadowExecutionEnabled")
+
     val message  = MaterializeWorkflowDescriptorActor.MaterializeWorkflow(workflowId, source, config)
     val materializeWorkflowDescriptorActor = context.actorOf(MaterializeWorkflowDescriptorActor.props())
     materializeWorkflowDescriptorActor.askNoTimeout(message) map {
       case MaterializationSuccess(descriptor) =>
-        val wfActor = context.actorOf(WorkflowActor.props(descriptor), s"WorkflowActor-$workflowId")
+        val wfActor = shadowExecutionEnabled match {
+          case true => context.actorOf(ShadowWorkflowActor.props(), s"WorkflowActor-$workflowId")
+          case _ => context.actorOf(WorkflowActor.props(descriptor), s"WorkflowActor-$workflowId")
+        }
```

Comment:
I'm assuming the plan is to refactor both the "classic" WorkflowActor as well as the new shadow WorkflowActor to manage their own validation since the classic/shadow distinction is more about interfacing with the backend workflow actors than validation?

If so this could pass the `WorkflowDescriptor` to the shadow `props` method to keep these two symmetric and the above can be simplified:

```scala
val buildProps = if (shadowExecutionEnabled) ShadowWorkflowActor.props else WorkflowActor.props
context.actorOf(buildProps(descriptor), s"WorkflowActor-$workflowId")
```